bootstrap_modal_forms/mixins.py

```diff
@@ -1,25 +1,33 @@
+from typing import TypeVar, Any
+
 from django.contrib import messages
 from django.contrib.auth import login as auth_login
+from django.contrib.auth.forms import AuthenticationForm
+from django.db.models import Model
 from django.http import HttpResponseRedirect
+from django.http.request import HttpRequest
+
+from .utils import *
 
-from .utils import is_ajax
+AuthForm = TypeVar('AuthForm', bound=AuthenticationForm)
```

Comment:
It would be better to define all the types in separate file.

Reply:
You need these types explicitly just in this module `mixins.py`. Are you sure to create another module and shifting these types into this new module, just to import them here anyways?